### DISCUSSION TITLE
Replicate selected tables from source

### DISCUSSION THREAD

**Discussion by csarsonas** [osalvador/ReplicaDB#87] [Q&A] [answered]
Good day sir.
I would like to replicate selected tables in the source DB.
Checking on the option-file, it seems that it is a 1:1 table - filename ratio (am i correct?).
What would be the best approach to execute a list of tables to be replicated?
In addition also, can this be executed every 2 minutes to be able to achieve at least real-time replication?
I'm currently using ReplicaDB via windows.

**Comment by osalvador** [comment, marked as answer]
Hi @csarsonas,

We are working on this feature, at the moment, you can create script for multiple table replication, see this comment: https://github.com/osalvador/ReplicaDB/issues/27#issuecomment-862229297 . Also you can use environment variables for replace the  table names.

ReplicaDB does not have a scheduler implemented, so on Windows a Scheduled Tasks must be used.

**Reply by csarsonas** [reply]
Thank you for your response @osalvador
